cookbook/catalog/common/validate.rst

```diff
@@ -0,0 +1,27 @@
+How to Validate Non-Product Objects
+===================================
+
+To validate an object, we use a classic Symfony Validator with Constraints defined in yaml files.
+
+Use the Validator
+-----------------
+
+The validator is a service, you can fetch it from the container
+
+.. code-block:: php
+
+    $validator = $this->getContainer()->get('validator');
+
+When you can validate an attribute, a family or another object, all business validation are applied.
```

Comment:
I don't get the "When you **_can_** validate (...)", why not "When you validate an attribute (...)"?